run.py

```diff
@@ -98,6 +104,11 @@ def run_one_step(func, nwarmup=WARMUP_ROUNDS, model_flops=None, num_iter=10, mod
             func()
             t1 = time.time_ns()
             result_summary.append([(t1 - t0) / 1_000_000])
+        if stress != 0:
+            cur_time = time.time_ns()
+            print(cur_time)
```

Comment:
Can we have a more informative output? E.g., `f"Iteration {num_iter}  |  {latency of this iter} ms | {estimated_rest_time}" `?

Reply:
fixed by https://github.com/pytorch/benchmark/pull/1002/commits/40efc6d4630c3098648f122e8abf03fdac60e111

Reply:
@FindHao sounds good, we can print number of iterations every 10 seconds (actually, let's change the unit of stress test to seconds instead of minutes)

Reply:
ok. 
I'll add the qps thing in next PR to avoid making this PR too messy.

Reply:
fixed in https://github.com/pytorch/benchmark/pull/1002/commits/e4d3fe423d9106cd9144a3700dd76cb698d1321f